presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -149,6 +149,7 @@
     private int s3SelectPushdownMaxConnections = 500;
 
     private boolean isTemporaryStagingDirectoryEnabled = true;
+    private String temporaryStagingDirectoryLocation = "/tmp/presto-${USER}";
```

Comment:
Why don't we use `path` instead of `location` for these configs/sessions/variables, etc.?